test/OAuth2/Storage/MongoDBTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace OAuth2\Storage;
+
+/**
+ * Description of MongoDBTest
+ *
+ * @author Roman Shuplov  <astronin@gmail.com>
+ */
+class MongoDBTest extends BaseTest 
```

Comment:
The tests for the Storage Interface methods are [here](https://github.com/bshaffer/oauth2-server-php/tree/develop/test/OAuth2/Storage), so I don't think any of these tests are needed.